### PR TITLE
feat: Adding info to Mac section

### PR DESCRIPTION
**Pull request by jcasman**
Intended to be merged with react-native.md file - Currently not complete but react-native-mac is not linked to the live document
Based on info from theta-client React Native tutorial and demo video

### CODE REVIEW

**Review by codetricity — CHANGES_REQUESTED** [review]
Make clickable link

```markdown
[https://github.com/ricohapi/theta-client/tree/main/react-native](https://github.com/ricohapi/theta-client/tree/main/react-native)
```

image

File: docs/react-native-mac.md [hunk 1]
```diff
@@ -2,4 +2,31 @@
 
 ## Overview
 
-This section will cover the basics of installing React Native of a Mac.
+This section will cover the basics of installing React Native on macOS. 
+It will also cover the preparation of the react-native demo.
+
+Use the README.md file here: https://github.com/ricohapi/theta-client/tree/main/react-native
```
[the remark this answers is not in the document]
Comment (jcasman) [reply docs/react-native-mac.md]: Fixed syntax. Checked locally with mkdocs serve.

**Review by codetricity — APPROVED** [review]

**Comment by codetricity** [comment]
make more progress and make another pull request

**Comment by jcasman** [comment]
I am currently getting this message when I look to do a `git checkout main` and then a `git pull main`

`MacBook-Air-5:docs jcasman$ git pull
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
SHA256:uNiVztksCsDhcc0u9e8BujQXVUpKZIDTMczCvj3tD2s.
Please contact your system administrator.
Add correct host key in /Users/jcasman/.ssh/known_hosts to get rid of this message.
Offending RSA key in /Users/jcasman/.ssh/known_hosts:1
Host key for github.com has changed and you have requested strict checking.
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.`

If I can solve this, I will. In the meantime, I'm working on local edits.

**Comment by jcasman** [comment]
I ran `ssh-keygen -R github.com` which appears to have fixed this problem.

**Comment by jcasman** [comment]
It looks like a GitHub mistake caused the problem: https://www.theregister.com/2023/03/24/github_changes_its_ssh_host/

**Comment by codetricity** [comment]
I'm waiting for another pull request.

**Pull request merged**